downgrade

```diff
@@ -75,12 +75,14 @@ search_packages() {
   ((NOARM)) || \
     curl --fail --silent "$index" | sed '
       /.* href="\('"$1"'.*\(any\|'"$ARCH"'\)\.pkg\.tar\.xz\)".*/!d;
-      s||'"$index"'\1|g; s|+| |g; s|%|\\x|' | xargs -0 printf "%b"
+      s||'"$index"'\1|g; s|+| |g; s|%|\\x|' | xargs -0 printf "%b" | grep "\-$version"
```

Comment:
I'd use `grep -F` here to avoid things in `$version` being treated like a regex.